docs/editor/artificial-intelligence.md

```diff
@@ -90,6 +90,14 @@ Copilot will provide a suggestion like the following:
 
 Use `kbstyle(Tab)` to accept the suggestion.
 
+### Alternative and partial solutions
+
+For any given input, GitHub Copilot may offer multiple suggestions. When Copilot offers a suggestion, you can hover over the suggestion to see the GitHub Copilot command palette for choosing suggestions:
```

Comment:
Instead of "GitHub Copilot command palette", let's call this "inline suggestion toolbar" or "ghost text toolbar"

Reply:
Makes sense! Used "inline suggestion toolbar" since I think we've used that terminology in spots like release notes.